docs/basics/user-interface/styling/container-queries.md

```diff
@@ -0,0 +1,163 @@
+---
+id: container-queries
+title: Container Queries
+---
+
+# Container Queries
```

Comment:
Can you add MinVersion here?
Like here https://github.com/AvaloniaUI/avalonia-docs/blob/775e9cf44d5c72305f5c25f83e39926015d2f766/docs/concepts/services/activatable-lifetime.md?plain=1#L6

Reply:
Added